pom.xml

```diff
@@ -610,40 +610,44 @@
       <plugins>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.10.1</version>
+          <version>3.11.0</version>
+        </plugin>
+        <plugin>
+          <artifactId>maven-install-plugin</artifactId>
+          <version>2.5.2</version>
```

Comment:
Here I had to *downgrade* the version of the maven-install-plugin to fix the build with newer Maven versions. See reasoning above.